### DISCUSSION TITLE
Add build instructions for the dum in github

### DISCUSSION THREAD

**Discussion by AchievedZeus574** [Docile-Alligator/Infinity-For-Reddit#1522] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

Given that Reddit will make x number of API calls cost how ever much money, users could build their own app with their own API key. Not everyone can 100% figure this out (like myself, but I can figure out google), it would be helpful if users had a loose guide on how to build the apk with gradle.

### Additional context

This would be great for users who want to figure it out but don't know where to start.

### How important is this feature?

Nice to have

**Comment by SirenGlitch** [comment]
[Found](https://colab.research.google.com/drive/13AE8RvjnCfuBJGaACEqxeBIMo33_l-Sc?usp=sharing#scrollTo=ViYXGNh0ytup)
